canarytokens/utils.py

```diff
@@ -66,5 +66,30 @@ def wrapper(*args, **kwargs):  # type: ignore
     return inner
 
 
-def token_type_as_readable(token_type: TokenTypes) -> str:
-    return token_type.replace("_", " ").lower().title()
+def token_type_as_readable(token_type: TokenTypes) -> tuple[str, str]:
```

Comment:
Instead of embedding the list of token types here (which will be fragile and subject to future issues), an alternative is to assume the article is "A", and have an override list in models.py for those that use "An". That means the worst break for new tokens is the wrong article.